src/game/clock.c

```diff
@@ -2,6 +2,9 @@
 
 #include "specific/s_clock.h"
 
+#include <stdio.h>
+#include <windows.h>
```

Comment:
Yh no, we don't include `<windows.h>` outside `specific/`. We previously used libc for this. Please restore the previous version without the milliseconds.

Reply:
Well I added ms to avoid adding in a duplicate name digit. This way the name is all made in `Shell_GetScreenshotName`. But ya it's Windows specific. This is so annoying lol. I'll see if there's a better way.